### ISSUE TITLE
replaceConsole option not taken into account

### ISSUE THREAD

**Issue by xuv** [jstty/stumpy#1]
in Node:

```
var Stumpy = require('stumpy');

var stumpy = Stumpy({
    replaceConsole: false
});

console.warn('Just a console warning');
stumpy.warn('Just a stumpy warning');
```

**Comment by jstty** [comment]
fixed in 3138b35 (v0.6.10)
Thanks for find this. I've added a task to make the tests better, to prevent this in the future.

**Comment by xuv** [comment]
Thanks for fixing this so quickly :)
